task/deprecated-image-check/0.2/deprecated-image-check.yaml

```diff
@@ -36,6 +36,9 @@ spec:
           value: $(params.BASE_IMAGES_DIGESTS)
       script: |
         #!/usr/bin/env bash
+        set -euo pipefail
+        trap 'echo "Script errored at line ${LINENO}: ${BASH_COMMAND}"' ERR
```

Comment:
Should be handle_error here?

Reply:
Actually we don't need it because we should not fail in this step